spec/components/personas/view_preview.rb

```diff
@@ -0,0 +1,9 @@
+module Persona
+  class ViewPreview < ViewComponent::Preview
```

Comment:
What's the purpose of these `ViewPreview` files given that it's not a spec file?

Reply:
https://dfe-twd-rttd-pr-151.herokuapp.com/view_components

i think you broken this now....

Reply:
have a peek, before it bolts

https://dfe-twd-rttd-pr-154.herokuapp.com/view_components/

Reply:
I understand. Will push fix.